cluster-service/config.tmpl.mk

```
@@ -11,6 +11,7 @@ ACR_NAME ?= {{ .svcAcrName }}
 OCP_ACR_NAME ?= {{ .ocpAcrName }}
 AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID ?= {{ .firstPartyAppClientId }}
 FPA_CERT_NAME ?= firstPartyCert
+MOCK_FPA_CERT_NAME ?= armHelperCert
```

Does this have any relation with armHelperIndentityCertificateBundle ?

---

The variable name seems to also be somewhat incorrect. The name of the variable is MOCK_FPA_CERT_NAME but the value is armHelperCert which seems related to the ARM Helper and not to the FPA

---

This name was taken from [this vault](https://portal.azure.com/#view/Microsoft_Azure_KeyVault/ListObjectVersionsRBACBlade/~/overview/objectType/certificates/objectId/https%3A%2F%2Faro-hcp-dev-svc-kv.vault.azure.net%2Fcertificates%2FarmHelperCert/vaultResourceUri/%2Fsubscriptions%2F1d3378d3-5a3f-4712-85a1-2485495dfc4b%2FresourceGroups%2Fglobal%2Fproviders%2FMicrosoft.KeyVault%2Fvaults%2Faro-hcp-dev-svc-kv/vaultId/%2Fsubscriptions%2F1d3378d3-5a3f-4712-85a1-2485495dfc4b%2FresourceGroups%2Fglobal%2Fproviders%2FMicrosoft.KeyVault%2Fvaults%2Faro-hcp-dev-svc-kv)

---

Perhaps we could align it somehow to what it does.

```suggestion
ARM_HELPER_CERT_NAME ?= armHelperCert
```

---

bumping this as it hasn't been fully taken onto account